### ISSUE TITLE
Why is the Vimeo modal width hard-coded?

### ISSUE THREAD

**Issue by joemaller** [ashleydw/lightbox#247]
Is there some reason why the Vimeo modals have their size hard-coded to 500px [(line 378)](https://github.com/ashleydw/lightbox/blob/ffcc8d1bbbd55cdce8ab3cacf4be05dbbae81e67/ekko-lightbox.js#L378)? Both `_showYoutubeVideo` ([line 367](https://github.com/ashleydw/lightbox/blob/ffcc8d1bbbd55cdce8ab3cacf4be05dbbae81e67/ekko-lightbox.js#L367))  and `_showInstagramVideo` ([line 385](https://github.com/ashleydw/lightbox/blob/ffcc8d1bbbd55cdce8ab3cacf4be05dbbae81e67/ekko-lightbox.js#L385)) use the data-width value before falling back.

Am I missing something or might this just be an oversight?

```js
_showVimeoVideo(id, $containerForElement) {
	let width = 500
	let height = this._$element.data('height') ||  width / ( 560/315 )
	return this._showVideoIframe(id + '?autoplay=1', width, height, $containerForElement)
}
```

**Comment by Turqueso** [comment]
I think it's for a minimum/default size, you can change it in options or with data-width

**Comment by joemaller** [comment]
@Turqueso Doesn't work. Vimeo's width is hard-coded to 500px. If we change line 378 to match the other two it works:

```js
	let width = this._$element.data('width') || 612
```

**Comment by joemaller** [comment]
I'm testing this to see if there's something I'm not understanding. Minor as it is, I'll have a PR shortly.

**Comment by joemaller** [comment]
Actually it's already there. #239 is exactly what I was going to do. Can we get that merged?

**Comment by joemaller** [comment]
Fixed in 92693291cc9c1cc52d085b1. Thanks!

**Comment by ashleydw** [comment]
👍 Once I've figured out #244 I'll release this fix
